PRChecker/Shared/ViewModels/PRListViewModel.swift

```diff
@@ -16,13 +16,13 @@ class PRListViewModel: ObservableObject {
     
     private var subscriptions = Set<AnyCancellable>()
     
-    func getPRList() {
+    func getPRList(completion: (() -> Void)? = nil)  {
         NetworkSerivce.shared.getAllPRs()
             .sink { error in
                 // TODO: Handle Errors
             } receiveValue: { prList in
                 DispatchQueue.main.async {
-                    self.prList = prList
+                    self.prList = prList.sorted { $0.rawCreatedAt > $1.rawCreatedAt }
```

Comment:
If we can get `updatedAt` for status(commented, approved..) changes, it would probably be better than `createdAt`.

Reply:
OK, let me swap it